lua/anywise_reg/keybinds.lua

```diff
@@ -54,22 +54,30 @@ local function setup_paste_keymaps(prefix)
     set_keymap(lhs, rhs)
 end
 
-local function setup_keymaps_for_prefix(prefix)
-    setup_yank_keymaps(prefix)
-    setup_paste_keymaps(prefix)
+local function setup_paste_behind_keymaps(prefix)
```

Comment:
Same comment as for the handler, I think this could take in a `operator` argument